fleet/pb/fleet.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
did you add this to the `proto` target in Makefile? policies is already there. I'm not sure why there is both a generated `fleet.pb.go` and `fleet_grpc.pb.go`, is one old maybe?